### PR TITLE
Translate permission strings. (Fixes #1895)

### LINKED ISSUE #1895: Change display name of permissions
Change the display name of all permissions to simpler string. Use don't know about our app-model system. So please use the third string only - like in 1.7.

Example:
"agenda | item | Can manage agenda" ->  "Can manage agenda"

Maybe we keep an additional 'group display name' string, e.g. 'Agenda', for better grouping in a matrix later.

And how we should translate these strings? Should the client do this?

### PR DESCRIPTION
**Pull request by emanuelschuetze**
Remove unused gettextCatalog content provider.

### CODE REVIEW

File: openslides/users/static/js/users/site.js [hunk 1]
```diff
@@ -872,8 +871,47 @@ angular.module('OpenSlidesApp.users.site', ['OpenSlidesApp.users'])
             $scope.closeThisDialog();
         };
     }
+])
+
+// Mark all permission strings for translation in JavaScript.
+// (see models.py of each djang app)
```
Comment (normanjaeckel) [inline openslides/users/static/js/users/site.js]: `Django`

**Comment by normanjaeckel** [comment]
Do you want to remove the translation tags for the permissions on server side (ugettext_noop in every models.py)?

**Comment by ostcar** [comment]
I don't get it? Do you want to get rid of the django translation file? What is the goal of this pull request?

File: openslides/users/static/js/users/site.js (hunk 1 above)
Comment (emanuelschuetze) [inline openslides/users/static/js/users/site.js]: done

**Comment by emanuelschuetze** [comment]
Removed all ugettext_noop from models.py.
All done. Ready for merge.

**Comment by normanjaeckel** [comment]
Travis failed. Otherwise looks good.

**Comment by emanuelschuetze** [comment]
@normanjaeckel Any idea what's wrong with the travis check?

File: openslides/core/models.py [hunk 2]
```diff
@@ -184,7 +183,7 @@ class Meta:
         ordering = ('name',)
         default_permissions = ()
         permissions = (
-            ('can_manage_tags', ugettext_noop('Can manage tags')),)
+            ('can_manage_tags', 'Can manage tags'))
```
Comment (normanjaeckel) [inline openslides/core/models.py]: Add comma before last closing bracket.

File: openslides/core/models.py [hunk 3]
```diff
@@ -204,7 +203,7 @@ class ConfigStore(models.Model):
     class Meta:
         default_permissions = ()
         permissions = (
-            ('can_manage_config', ugettext_noop('Can manage configuration')),)
+            ('can_manage_config', 'Can manage configuration'))
```
Comment (normanjaeckel) [inline openslides/core/models.py]: Here the same.

File: openslides/core/models.py [hunk 4]
```diff
@@ -230,7 +229,7 @@ class ChatMessage(RESTModelMixin, models.Model):
     class Meta:
         default_permissions = ()
         permissions = (
-            ('can_use_chat', ugettext_noop('Can use the chat')),)
+            ('can_use_chat', 'Can use the chat'))
```
Comment (normanjaeckel) [inline openslides/core/models.py]: Here the same.

**Comment by emanuelschuetze** [comment]
Fixed and rebased.

**Pull request merged**